ios/SmartSyncExplorerReactNative.xcodeproj/project.pbxproj

```diff
@@ -295,6 +429,7 @@
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				CODE_SIGN_ENTITLEMENTS = SmartSyncExplorerReactNative/SmartSyncExplorerReactNative.entitlements;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = "";
```

Comment:
You might have to manually remove the `DEVELOPMENT_TEAM` line with a text editor.